cypress/integration/common/index.js

```diff
@@ -198,11 +198,15 @@ And('there should be no {string} table in the document', (string) => {
 And(
 	'the Patient Information link to {string} appears on the page',
 	(patientInfoLink) => {
-		cy.get('a.definition-link').should('have.attr', 'href', patientInfoLink);
+		cy.get('a.patient-information-button').should(
+			'have.attr',
+			'href',
+			patientInfoLink
+		);
```

Comment:
The only thing I would add here, considering the redesign is another assertion of a text for screen reader:
cy.get('a.patient-information-button').should('have.text',buttonText);